src/tests/JIT/opt/Tailcall/TailcallVerifyWithPrefix.il

```diff
@@ -339,146 +339,6 @@
-    IL_0097:  call       void [System.Console]System.Console::WriteLine(string)
-    IL_009c:  ldsfld     int32 TailcallVerify.Condition17::Result
-    IL_00a1:  ret
-  } // end of method Condition17::Test1
-
-  .method private hidebysig static void  Caller1(valuetype TailcallVerify.Unsafe u) cil managed
-  {
-    // Code size       74 (0x4a)
-    .maxstack  2
-    IL_0000:  ldc.i4.0
-    IL_0001:  newobj     instance void [mscorlib]System.Diagnostics.StackFrame::.ctor(bool)
-    IL_0006:  callvirt   instance class [mscorlib]System.Reflection.MethodBase [mscorlib]System.Diagnostics.StackFrame::GetMethod()
-    IL_000b:  callvirt   instance string [mscorlib]System.Reflection.MemberInfo::get_Name()
-    IL_0010:  ldstr      "Caller"
-    IL_0015:  callvirt   instance int32 [mscorlib]System.String::IndexOf(string)
-    IL_001a:  ldc.i4.m1
-    IL_001b:  bne.un.s   IL_002e
-
-    IL_001d:  ldstr      "Failed, Method was inlined..."
-    IL_0022:  call       void [System.Console]System.Console::WriteLine(string)
-    IL_0027:  ldc.i4.s   101
-    IL_0029:  stsfld     int32 TailcallVerify.Condition17::Result
-    IL_002e:  ldarga.s   u
-    IL_0030:  ldflda     valuetype TailcallVerify.Unsafe/'<data>e__FixedBuffer0' TailcallVerify.Unsafe::data
-    IL_0035:  ldflda     uint8 TailcallVerify.Unsafe/'<data>e__FixedBuffer0'::FixedElementField
-    IL_003a:  conv.u
-    IL_003b:  volatile.
-    IL_003d:  ldsfld     int32 modreq([mscorlib]System.Runtime.CompilerServices.IsVolatile) TailcallVerify.Condition17::zero
-    IL_0042:  conv.u1
-    IL_0043:  stind.i1
-    IL_0044:  tail. call       void TailcallVerify.Condition17::Callee1()
-    IL_0049:  ret
-  } // end of method Condition17::Caller1
-
-  .method private hidebysig static void  Callee1() cil managed noinlining
-  {
-    // Code size       11 (0xb)
-    .maxstack  8
-    IL_0000:  ldc.i4.1
-    IL_0001:  volatile.
-    IL_0003:  ldsfld     int32 modreq([mscorlib]System.Runtime.CompilerServices.IsVolatile) TailcallVerify.Condition17::zero
-    IL_0008:  div
-    IL_0009:  pop
-    IL_000a:  ret
-  } // end of method Condition17::Callee1
-
-  .method public hidebysig specialname rtspecialname
-          instance void  .ctor() cil managed
-  {
-    // Code size       7 (0x7)
-    .maxstack  8
-    IL_0000:  ldarg.0
-    IL_0001:  call       instance void [mscorlib]System.Object::.ctor()
-    IL_0006:  ret
-  } // end of method Condition17::.ctor
-
-  .method private hidebysig specialname rtspecialname static
-          void  .cctor() cil managed
-  {
-    // Code size       16 (0x10)
-    .maxstack  8
-    IL_0000:  ldc.i4.0
-    IL_0001:  volatile.
-    IL_0003:  stsfld     int32 modreq([mscorlib]System.Runtime.CompilerServices.IsVolatile) TailcallVerify.Condition17::zero
-    IL_0008:  ldc.i4.s   100
-    IL_000a:  stsfld     int32 TailcallVerify.Condition17::Result
-    IL_000f:  ret
-  } // end of method Condition17::.cctor
-
-} // end of class TailcallVerify.Condition17
-
```

Comment:
This particular test was relying on no tailcall to happen from method with unsafe local in it.